modules/sff/module/src/sff.c

```diff
@@ -357,6 +372,9 @@ sff_media_type_get(sff_module_type_t mt)
         case SFF_MODULE_TYPE_1G_BASE_ZX:
         case SFF_MODULE_TYPE_100_BASE_LX:
         case SFF_MODULE_TYPE_100_BASE_FX:
+        case SFF_MODULE_TYPE_100G_BASE_LR:
```

Comment:
You should group the 100G enums together.

Reply:
grouped 100G...